spec/models/projects/allowed_to_scope_spec.rb

```diff
@@ -137,72 +138,108 @@
       it_behaves_like 'is empty'
     end
 
-    context 'w/ the user being member
-             w/ the permission being public' do
+    context 'with the user being member
+             with the permission being public' do
       before do
         member.save!
       end
 
       it 'includes the project' do
-        expect(Project.allowed_to(user, public_action)).to match_array [project]
+        expect(described_class.allowed_to(user, public_action)).to contain_exactly(project)
       end
     end
 
-    context 'w/ the user being member
-             w/ the permission being public
-             w/o the associated module being active' do
+    context 'with the user being member
+             with the permission being public
+             without the associated module being active' do
       before do
         member.save!
         project.enabled_modules = []
       end
 
       it 'is empty' do
-        expect(Project.allowed_to(user, public_action)).to be_empty
+        expect(described_class.allowed_to(user, public_action)).to be_empty
       end
     end
 
-    context 'w/ the user being member
-             w/ the permission being public and not module bound
-             w/ no module bing active' do
+    context 'with the user being member
+             with the permission being public and not module bound
+             with no module being active' do
       before do
         member.save!
         project.enabled_modules = []
       end
 
       it 'includes the project' do
-        expect(Project.allowed_to(user, public_non_module_action)).to match_array [project]
+        expect(described_class.allowed_to(user, public_non_module_action)).to contain_exactly(project)
+      end
+    end
+
+    context 'with the user being member
+             with the permission being module bound
```

Comment:
```suggestion
             with the permission not being module bound
```

Reply:
You are right, @aaron-contreras. I'll fix it in both descriptions